docs/conf.py

```diff
@@ -279,7 +279,7 @@
     'python': ('https://docs.python.org/3', None),
     'sphinx': ('https://www.sphinx-doc.org/en/master', None),
     'tox': ('https://tox.readthedocs.io/en/latest', None),
-    'twisted': ('https://twistedmatrix.com/documents/current', None),
+    'twisted': ('https://twistedmatrix.com/documents/current/api', None),
```

Comment:
Try simply adding a `'twistedapi': ('https://twistedmatrix.com/documents/current/api', None),` line.

Reply:
Great, that seems to work. Thanks!